eng/Build.props

```diff
@@ -23,6 +23,7 @@
                $(RepoRoot)src\Http\Routing\test\testassets\RoutingSandbox\*.*proj;
                $(RepoRoot)src\Security\Authentication\Negotiate\test\testassets\Negotiate.Client\*.*proj;
                $(RepoRoot)src\Security\Authentication\Negotiate\test\testassets\Negotiate.Server\*.*proj;
+               $(RepoRoot)src\Grpc\HttpApi\**\*.csproj
```

Comment:
Unfortunately this will do the opposite of what you want because you're in double negative territory. You want to _include_ `$(RepoRoot)src\Grpc\HttpApi\**\*.csproj` e.g.
```suggestion
```

and add `$(RepoRoot)src\Grpc\HttpApi\**\*.csproj` to the `Include` attribute here. Could also do the addition to `@(ProjectToExclude)` in a separate item because the projects aren't "meant to be referenced only by tests".